agent/src/agent_utils_win.cc

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 The Chromium Authors.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <windows.h>
+
+#include "content_analysis/sdk/result_codes.h"
+
+namespace content_analysis {
+namespace sdk {
+
+#define ERR_TO_RC(ERR, RC) case ERR: return ResultCode::RC;
+
+ResultCode ErrorToResultCode(DWORD err) {
+  switch (err) {
```

Comment:
`case X: return Y;` is 17 chars while `ERR_TO_RC(X, Y);` is 16 chars.
That means all this macro really shortens is the ResultCode prefxi, instead we can just use `using ResultCode` in the function scope?

Reply:
I can't get that to compile.  It does not seem like the `using` keyword works with enums.  I could say `using RC = ResultCode` but then I don't think that helps much.  Am I missing something?

Reply:
After discussion, I've left it as is.